graphdatascience/aura_graph_data_science.py

```diff
@@ -26,43 +26,41 @@ def __init__(
         endpoint: str,
         auth: Tuple[str, str],
         aura_db_connection_info: AuraDbConnectionInfo,
-        database: Optional[str] = None,
         arrow_disable_server_verification: bool = True,
         arrow_tls_root_certs: Optional[bytes] = None,
         bookmarks: Optional[Any] = None,
     ):
-        gds_query_runner = Neo4jQueryRunner.create(endpoint, auth, aura_ds=True)
-        self._driver_config = gds_query_runner.driver_config()
+        gds_query_runner = ArrowQueryRunner.create(
+            Neo4jQueryRunner.create(endpoint, auth, aura_ds=True),
+            auth,
+            True,
+            arrow_disable_server_verification,
+            arrow_tls_root_certs,
+        )
+
+        gds_query_runner.set_database("neo4j")
```

Comment:
This is not necessary as this is the default

Reply:
i dont see where this is the default. The default is `None`.

Reply:
https://github.com/neo4j/graph-data-science-client/blob/main/graphdatascience/query_runner/neo4j_query_runner.py#L83

Reply:
Still necessary as the default is `None` and not `neo4j` which is a bit counter-intuitive. I added a comment clarifying this